pkg/keymanagementprovider/azurekeyvault/types/types.go

```diff
@@ -25,6 +25,8 @@ const (
 	StatusName = "Name"
 	// Certificate version string for the certificate status property
 	StatusVersion = "Version"
+	// Enabled string for the certificate status property
+	StatusEnabled = "True"
```

Comment:
Should this be `Enabled` instead?

Reply:
Good catch @shahramk64! Yes, it should be.